pages/common/gnomon.md

```diff
@@ -15,10 +15,10 @@
 
 `{{npm test}} | gnomon --type=absolute`
 
-- Set a high threshold of 0.5 seconds for the elapsed time; exceeding which the timestamp will be colored bright red:
+- Use a 0.5 second high threshold for the elapsed time; exceeding which the timestamp will be colored bright red:
 
-`{{npm test}} | gnomon --high {{0.5}}`
+`{{npm test}} | gnomon --high 0.5`
 
-- Set a medium threshold of 0.2 seconds (Timestamp will be colored bright yellow):
+- Use a medium 0.2 seconds threshold (timestamp will be colored bright yellow):
```

Comment:
You changed the meaning of this sentence.

"Use a medium threshold of 0.2 seconds" => Set the ***medium threshold*** to a value of 0.2 seconds
"Use a medium 0.2 seconds threshold" => Set the threshold to a ***medium value*** of 0.2 seconds

```suggestion
- Use a medium threshold of 0.2 seconds for the elapsed time, exceeding which the timestamp will be colored bright yellow:
```